test/test_unit_credential_store_factory.py

```diff
@@ -0,0 +1,31 @@
+# Builtin
+import unittest
+from unittest.mock import patch, call
+
+from conjur.logic.credential_provider import CredentialStoreFactory, KeystoreCredentialsProvider, \
+    FileCredentialsProvider
+from conjur.wrapper import KeystoreAdapter
+
+
+class CredentialStoreFactoryTest(unittest.TestCase):
+    @patch.object(KeystoreAdapter, "get_keyring_name", return_value="SecretService Keyring")
+    @patch.object(KeystoreAdapter, "is_keyring_accessible", return_value=True)
+    def test_create_credential_store_returns_keystore_credentials_provider_when_there_is_keyring_and_its_accessible(
+            self, mock_keystore_adapter, mock_another_keystore_adapter):
+        provider, message = CredentialStoreFactory.create_credential_store()
```

Comment:
if we don't care for the `message`, you can just write:
```suggestion
        provider, _ = CredentialStoreFactory.create_credential_store()
```

Reply:
Done